RFC/src/SUMMARY.md

```diff
@@ -7,6 +7,31 @@
   - [RFC-0110: Base nodes](RFC-0110_BaseNodes.md)
   - [RFC-0130: Mining](RFC-0130_Mining.md)
   - [RFC-0150: Wallets](RFC-0150_Wallets.md)
+- [RFC-0200: Tari-specific extensions to Mimblewimble](RFC-0200_BaseLayerExtensions.md)
+  - [RFC-0210: Namespace registration](RFC-0210_NamespaceRegistration.md)
+  - [RFC-0220: Asset checkpoints](RFC-0220_AssetCheckpoints.md)
+  - [RFC-0230: Asset multiparty contracts](RFC-0230_AssetContracts.md)
 - [RFC-0300: The Digital Assets Network](RFC-0300_DAN.md)
+- [RFC-0320: Validator Nodes](RFC-0320_ValidatorNodes.md)
+  - [RFC-0321: Asset discovery](RFC-0321_VNAssetDiscovery.md)
+  - [RFC-0322: Validator Node Registration](RFC-0322_VNRegistration.md)
+  - [RFC-0323: Validator Node committee selection](RFC-0323_VNCommittees.md)
   - [RFC-0310: Digital Asset templates](RFC-0310_Assets.md)
+  - [RFC-0325: The asset read-only API](RFC-0325_AssetReadAPI.md)
+- [RFC-0340: Validator Node Consensus](RFC-0340_VNConsensusOverview.md)
+  - [RFC-0341: Validator node instructions](RFC-0341_VNInstructions.md)
+  - [RFC-0342: The VN consensus algorithm](RFC-0342_VNConsensusAlgorithm.md)
+  - [RFC-0344: Asset transfer mechanism](RFC-0344_VNAssetTransfer.md)
+  - [RFC-0345: Asset retirement](RFC-0345_VNAssetRetirement.md)
+- [RFC-0350: DAN Dispute resolution mechanism](RFC-0350_DisputeResolution.md)
+- [RFC-0360: Tari Collections](RFC-0360_TariCollections.md)
+- [RFC-0370: Asset Managers](RFC-0370_AssetManager.md)
+  - [RFC-0371: Asset Key management](RFC-0371_AssetKeyManagement.md)
+  - [RFC-0372: Peer-to-peer communication for asset managers](RFC-0372_AMP2PCommunication.md)
+  - [RFC-0373: Instruction API](RFC-0373_InstructionAPI.md)
+  - [RFC-0374: Asset Creation and management](RFC-0374_AssetCreation.md)
+  - [RFC-0375: Tari wallet bridge](RFC-0375_AMWalletBridge.md)
+- [RFC-0500: Tari payment channels](RFC-0500_PaymentChannels.md)
```

Comment:
What about adding something for asset transactions/exchanges, i.e. Bob gives two tickets for two different events to Alice in exchange for an end-game item plus some Tari coins.

Reply:
I believe this is an import feature, I think we might want to leave this out till we have a mvp dan running and working on atleast testnet

Reply:
Agree. At the very least, barter can be simulated by applying multiple conventional sale transactions.

Reply:
So we can have an RFC about it then, irrespective of when it will be implemented